lib/chef/node/mixin/immutablize_hash.rb

```diff
@@ -129,6 +129,7 @@ module ImmutablizeHash
           values_at
           zip
           deep_merge
+          deep_merge?
```

Comment:
Because we're using the internal API of activesupport, and it changed between minor versions.